.gitignore

```diff
@@ -26,7 +26,7 @@
 /tmp/
 
 # Used by dotenv library to load environment variables.
-# .env
+ /.env
```

Comment:
maybe you accidentally added this after already committing the `.env` file to git?